### ISSUE TITLE
Ability to apply force to a player.

### ISSUE THREAD

**Issue by ghost** [INPStarfall/Starfall#118]
I would like to see the ability to apply a force onto a player entity.
You can do that with any other entity, so why would you block it for players ?
I think it should have some obvious limitations:
Only allow applying force to a player when you're the person that placed the gate but allow it on any player if admin.

(Same goes for setVelocity and other functions of it's kind, and also setAngles should set the player's eyeAngles)

**Comment by AlexanderArvidsson** [comment]
Would be cool if it had CPPI implementation as well, so you can applyforce people in your prop protect friends list.
For those unaware, CPPI is like the guidelines of PP systems, and it makes it so addons work with all kinds of PP systems.

**Comment by Xandaros** [comment]
Not CPPI, we have our own permissions system, which is still missing an interface, but it exists.
And FYI, we aren't blocking player, it just doesn't work. [Code](https://github.com/INPStarfall/Starfall/blob/develop/lua/starfall/libs_sv/entities.lua#L161)

**Comment by AlexanderArvidsson** [comment]
But then you won't be able to use i.e apply force on other players through pp systems, unless you are making a way to give certain player permission to do certain things, like applyforce.

**Comment by Xandaros** [comment]
> unless you are making a way to give certain player permission to do certain things

Didn't I say we already have a system and it is just lacking an interface? …

**Comment by AlexanderArvidsson** [comment]
Right, sorry, but I couldn't possibly know what is and what is not already implemented. The last time I heard about this permissions system was quite a while ago, and at that time you hadn't implemented it yet. Since then I had not heard anything else from it, so I didn't know if it was fully implemented or not.
